examples/audio/eeg_signal.py

```diff
@@ -0,0 +1,527 @@
+"""
+Title: EEG Signal Classification for Stimuli identification
```

Comment:
if possible lets avoid field specific acronyms in titles.

Maybe "Electroencephalography Signal Classification..."

or "Brain Signal Classification..."

Reply:
I'll make that change right away.